### PR TITLE
Added docs for the cache:pool:clear command

### LINKED ISSUE #7173: Add docs for the "cache:pool:clear" command
In https://github.com/symfony/symfony/pull/19891 Nicolas added a new `cache:pool:clear` command. Should we document it or add a note about it?

### PR DESCRIPTION
**Pull request by javiereguiluz**
This fixes #7173.

### CODE REVIEW

**Comment by chalasr** [comment]
Should we mention that the command allows to clear multiple cache pools at once?
Otherwise 👍

**Comment by javiereguiluz** [comment]
@chalasr in docs we usually just mention commands without explaining the details (same as our policy of not documenting the arguments of the methods). But if @xabbuh and @wouterj agrees with you, I'll update this. Thanks!

File: components/cache/cache_pools.rst [hunk 1]
```diff
@@ -278,4 +278,13 @@ when all items are successfully deleted)::
...
     $cacheIsEmpty = $cache->clear();
 
+.. tip::
+
+    If the Cache component is used inside a Symfony application, you can remove
+    all the items of a given cache pool with the following command:
+
+    .. code-block:: terminal
+
+        $ ./bin/console cache:pool:clear <cache-pool-name>
```
Comment (wouterj) [inline components/cache/cache_pools.rst]: should be `php bin/console cache:pool:clear <cache-pool-name>` (this is the only Windows and Unix compatible command)

Comment (wouterj) [inline components/cache/cache_pools.rst]: We can use comments to document other usages (IIRC, we do it in other places as well):

```rst
# clears the "cache.app" pool
$ php bin/console cache:pool:clear cache.app

# clears the "cache.validation" and "cache.app" pool
$ php bin/console cache:pool:clear cache.validation cache.app
```

**Review by HeahDude — APPROVED** [review]

**Review by chalasr — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @javiereguiluz.

**Pull request merged**